firebase-abt/src/main/java/com/google/firebase/abt/FirebaseABTesting.java

```diff
@@ -350,14 +354,17 @@ private void throwAbtExceptionIfAnalyticsIsNull() throws AbtException {
    * breaking, or if the underlying Analytics clear method is failing.
    */
   private void removeExperimentFromAnalytics(String experimentId) {
-    analyticsConnector.clearConditionalUserProperty(
-        experimentId, /*clearEventName=*/ null, /*clearEventParams=*/ null);
+    AnalyticsConnector connector = this.analyticsConnector.get();
+    if (connector != null) {
+      connector.clearConditionalUserProperty(
+          experimentId, /*clearEventName=*/ null, /*clearEventParams=*/ null);
+    }
   }
 
   @WorkerThread
   private int getMaxUserPropertiesInAnalytics() {
     if (maxUserProperties == null) {
-      maxUserProperties = analyticsConnector.getMaxUserProperties(originService);
+      maxUserProperties = analyticsConnector.get().getMaxUserProperties(originService);
```

Comment:
Thinking: `@WorkerThread` methods don't need to check for null?

Reply:
The check is not required for the same reason you mentioned below, the check is done prior to calling this method